src/lib/core/WeaveConfig.h

```diff
@@ -2394,6 +2394,17 @@
 extern const char WEAVE_NON_PRODUCTION_MARKER[];
 #endif
 
+/**
+ * @def WEAVE_CONFIG_RESET_WEAVE_CONFIG_TIMEOUT_MSCES
+ *
+ * @brief The amount of time (in milliseconds) which a device send the respond
+ *    to servers and wait for its delivery before initiating the factory-reset.
```

Comment:
Recommend rephrasing as "The amount of time that a device waits after receiving a signal to reset its Weave configuration before executing the reset action."